apps/web/src/composables/auth.ts

```diff
@@ -33,7 +34,47 @@ export default function useAuth() {
         }
     }
 
+    function addAccount(provider: ProviderString, address: string) {
+        address = address.toLowerCase()
+        const localStorage = window.localStorage
+        const accounts = JSON.parse(localStorage.getItem('accounts') as string) || {}
+
+        // Check if the address already exists on a different added provider (remove if so)
+        for (const existingProvider in accounts) {
+            if (accounts[existingProvider].includes(address) && existingProvider !== provider) {
+                accounts[existingProvider] = accounts[existingProvider].filter((existingAddress: string) => existingAddress !== address)
```

Comment:
Nice solution, can you test if it plays well with Vue reactivity? Like with a UI list component.

Reply:
Ok looks good on the reactivity!

Reply:
Ok! Do you want to approve and merge? @shanejearley

Reply:
Resolved.